src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="Surname"
+          autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          Name:
+          <input
+          type="text"
+          name="Name"
+          maxlength="25"
+          autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          How old are You?
+          <input
+          type="number"
+          name="How old are You?"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </div>
+
+        <div class="form-field">
+          Full date of birth:
+          <input
+          type="date"
+          name="Full date of birth"
+          >
+        </div>
+
+        <div class="form-field">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="I accept the term of the agreement"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          E-mail:
+          <input
+          type="email"
+          name="E-mail"
+          placeholder="email@example.com"
+          >
+        </div>
+        <div class="form-field">
+          Password:
+          <input
+          type="password"
+          name="Password"
+          minlength="5"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form">
+
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="Yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+            type="radio"
+            name="No"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="form-field">
+          What is your favorite color?
+          <input
+          type="color"
+          name="color"
+          >
+        </div>
+
+        <div class="form-field">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          >
+        </div>
+
+        <div class="form-field">
+          What are your favorite brend of cars?
+          <label>
+            <select name="car" multiple>
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Jeep">Lada</option>
```

Comment:
The option text should match the value attribute. Change the option text to 'Lada' instead of 'Jeep'.